modules/mirror-registry-introduction.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-installation-images.adoc
+
+[id="mirror-registry-introduction_{context}"]
+== Mirror registry for {product-title} introduction
+
+Mirror registry allows users to install a small-scale version of Red Hat Quay and its required components using the `mirror-registry` command line interface (CLI) tool. Mirror registry comes equipped with pre-configured local storage and a local database. It also includes pre-configured user credentials and access permissions with a single set of inputs and no additional configuration choices to get started.
+
+Mirror registry provides a pre-determined network configuration and reports deployed component credentials and URIs upon success. A limited set of optional configuration inputs like fully qualified domain name (FQDN) services, superuser name and password, and custom TLS certificates are also provided. These services allow users to easily create an offline mirror of all {product-title} release content when running {product-title} in restricted network environments.
+
+Mirror registry is limited to hosting images that are required to install a disconnected {product-title} cluster. It uses local storage on your {op-system-base-full} machine, and storage supported by {op-system-base} is supported by mirror registry. Mirror registry is not supported for production load, and is intended for use with {product-title} images only.
+
```

Comment:
```suggestion
The mirror registry is based on a streamlined, minimal Red Hat Quay installation. Unlike Red Hat Quay, the mirror registry is not a highly available registry and only local filesystem storage is supported. Albeit technically possible, it is discouraged to use the the mirror registry with more than one cluster, to avoid create a single point of failure when updating your cluster fleet. It is advised to leverage the mirror registry to install a cluster which in turns hosts a production-grade, highly-available registry (such as Red Hat Quay) which serves {product-title} content to other clusters. Use of the mirror registry is optional if another container registry is already available in the install environment.
```

Reply:
Added, slightly altered.